docs/Getting Started/Fedora/Fedora Root On ZFS.rst

```diff
@@ -0,0 +1,818 @@
+.. highlight:: sh
+
+Fedora Root on ZFS
+==================
+
+.. contents:: Table of Contents
+  :local:
+
+Overview
+--------
+
+Caution
+~~~~~~~
+
+- This HOWTO uses a whole physical disk.
+- Do not use these instructions for dual-booting.
+- Backup your data. Any existing data will be lost.
+
+System Requirements
+~~~~~~~~~~~~~~~~~~~
+
+- `64-bit Fedora Workstation (spins are not yet tested yet)
+  <https://getfedora.org/en/workstation/download/>`__
+- `A 64-bit kernel is strongly encouraged.
+  <https://openzfs.github.io/openzfs-docs/Project%20and%20Community/FAQ.html#bit-vs-64-bit-systems>`__
+- Installing on a drive which presents 4 KiB logical sectors (a “4Kn” drive)
+  only works with UEFI booting. This not unique to ZFS. `GRUB does not and
+  will not work on 4Kn with legacy (BIOS) booting.
+  <http://savannah.gnu.org/bugs/?46700>`__
+- You MUST use a UEFI system for this guide as GRUB is not yet supported. To verify UEFI, go to /sys/firmware/efi/efivars and make sure it is not empty.
+
+Computers that have less than 2 GiB of memory run ZFS slowly. 4 GiB of memory
+is recommended for normal performance in basic workloads. If you wish to use
+deduplication, you will need `massive amounts of RAM
+<http://wiki.freebsd.org/ZFSTuningGuide#Deduplication>`__. Enabling
+deduplication is a permanent change that cannot be easily reverted.
+
+Support
+~~~~~~~
+
+If you need help, reach out to the community using the :doc:`zfs-discuss
+mailing list <../../Project and Community/Mailing Lists>` or IRC at
+`#zfsonlinux <irc://irc.freenode.net/#zfsonlinux>`__ on `freenode
+<https://freenode.net/>`__. If you have a bug report or feature request
+related to this HOWTO, please `file a new issue and mention @rlaager
```

Comment:
I don't use Fedora. Take my name out. If you're going to maintain this, you can sub in your username. Also, update the URL below for filing an issue, which has my username in it and also has the distro name and version (so if someone clicks the link and keeps the suggested text, I know which HOWTO we are talking about).

Reply:
Ok, fixed